services/auth/token/src/lambdas/generateToken.ts

```diff
@@ -9,8 +9,8 @@ import log from '../libs/logs';
 import Joi from 'joi';
 
 const CONFIG_AUTH_SECRETS = config.auth.secrets;
-const ACCESS_TOKEN_EXPIRES_IN_MINUTES = 20;
-const REFRESH_TOKEN_EXPIRES_IN_MINUTES = 30;
+const ACCESS_TOKEN_EXPIRES_IN_MINUTES = 60;
```

Comment:
Don't think that refresh token should expire just 10 minutes after the access token. In most cases the refresh token has a longer expiration time.

example:
access token expires after 60 minutes
refresh token expires after 24 hours